### DISCUSSION TITLE
v0.6 - The Tray Update

### DISCUSSION THREAD

**Discussion by U-C-S** [U-C-S/Hurl#35] [Announcements]
A Minor Update focusing on Single-Instancing, Minimizing to Tray. Might noticeably help in reducing startup times.
- Single-Instancing is made possible with https://github.com/soheilkd/SingleInstanceCore
- Minimize to Tray with Tray contextmenu options, powered by WPF-UI library

Development Updates include bit of Restructuring and Polishing to make code cleaner.
- Use https://github.com/U-C-S/SingleInstanceCore as git submodule
- Polish `GetBrowsers`, `SettingsFile` classes

**Full Changelog**: https://github.com/U-C-S/Hurl/compare/v0.5.0...v0.6.0

<hr /><em>This discussion was created from the release <a href='https://github.com/U-C-S/Hurl/releases/tag/v0.6.0'>v0.6 - The Tray Update</a>.</em>

**Comment by nitin2953** [comment]
Tray `Reload` button showing error

![image](https://user-images.githubusercontent.com/34852979/158873669-c0cd4394-76b5-43b3-8832-e267f052d8a2.png)

**Reply by U-C-S** [reply]
Looking into it. Even I'm getting it.
